docs/reference/commandline/build.md

```diff
@@ -29,7 +29,7 @@ parent = "smn_cli"
       -m, --memory=""                 Memory limit for all build containers
       --memory-swap=""                A positive integer equal to memory plus swap. Specify -1 to enable unlimited swap.
       --no-cache                      Do not use cache when building the image
-      --pull                          Always attempt to pull a newer version of the image
+      --pull=<content-trust>          Always attempt to pull a newer version of the image
```

Comment:
@dkjer I read your PR description. However I still don't understand what `--pull=<content-trust>` means here. Would you mind clarifying?